qsirecon/interfaces/amico.py

```diff
@@ -127,9 +129,24 @@ def _write_external_formats(self, runtime, fit_obj, mask_img, suffix):
 
 
 class NODDIInputSpec(AmicoInputSpec):
-    dPar = traits.Float(mandatory=True, desc="Parallel diffusivity")
-    dIso = traits.Float(mandatory=True, desc="Isotropic diffusivity")
-    isExvivo = traits.Bool(False, usedefault=True, desc="Whether the data is ex vivo")
+    dPar = traits.Float(
+        mandatory=True,
+        desc="Parallel diffusivity constant",
```

Comment:
```suggestion
        desc=(
            "Parallel diffusivity, in mm^2/s. "
            "Passed to AMICO as dPar. "
            "Defined in the recon spec as dPar. "
            "Stored in the metadata as ParallelDiffusivity."
        ),
```

In the interest of making it easier for users to see how parameters in the recon spec translate to metadata, function parameters, etc.